src/containers/PrimeStatusBanner/index.tsx

```diff
@@ -299,12 +306,32 @@ const PrimeStatusBanner: React.FC<PrimeStatusBannerProps> = props => {
 
   // TODO: wire up
   const isLoading = isGetIsAddressPrimeLoading;
-  const primeClaimWaitingPeriodSeconds = 90 * 24 * 60 * 60; // 90 days in seconds
-  const userStakedXvsTokens = new BigNumber('100');
-  const minXvsToStakeForPrimeTokens = new BigNumber('1000');
+
+  const { data: primeClaimWaitingPeriodSeconds } = useGetPrimeClaimWaitingPeriod();
+
+  const { data: userStakedXvsTokensData } = useGetXvsVaultUserInfo(
+    {
+      accountAddress: accountAddress || '',
+      poolIndex: 1,
```

Comment:
This will be environment-dependent. It might be worth asking the contract team what the best way for us is to get the index of the vault users will have to stake in for Prime: I think we'd be safer fetching this from a contract (perhaps the contract team could even come up with a new method on the Prime contract for us to check the user stake).